src/Maestro/Maestro.Web.Tests/DefaultChannelsController20200220Tests.cs

```diff
@@ -0,0 +1,338 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Maestro.Data;
+using Maestro.Web.Api.v2020_02_20.Controllers;
+using Maestro.Web.Api.v2020_02_20.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.DotNet.DarcLib;
+using Microsoft.DotNet.Internal.Testing.Utility;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Hosting.Internal;
+using Microsoft.Extensions.Internal;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using static Maestro.Web.Api.v2020_02_20.Models.DefaultChannel;
+
+namespace Maestro.Web.Tests
+{
+    [TestFixture]
+    public class DefaultChannelsController20200220Tests
+    {
+        [Test]
+        public async Task CreateAndGetDefaultChannel()
+        {
+            using TestData data = await BuildDefaultAsync();
+            string channelName = "TEST-CHANNEL-LIST-REPOSITORIES";
+            string classification = "TEST-CLASSIFICATION";
+            string repository = "FAKE-REPOSITORY";
+            string branch = "FAKE-BRANCH";
+
+            Channel channel1, channel2;
+            {
+                var result = await data.ChannelsController.CreateChannel($"{channelName}-1", classification);
+                channel1 = (Channel)((ObjectResult)result).Value;
```

Comment:
[nit] Looks like our style is to leave a space after the cast.

```suggestion
                channel1 = (Channel) ((ObjectResult) result).Value;
```

Reply:
turns out I had to update my editor to be able to do this, done.